src/features/Novel/components/NovelTabs/NovelTabs.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { FC, memo } from 'react';
+import { NavLink } from 'react-router-dom';
+import styles from './NovelTabs.module.css';
+
+const tabNames = [
```

Comment:
Should we move it to another file and change to array of objects of this kind `{ name, link }`.